examples/configs/distillation_math.yaml

```
@@ -0,0 +1,170 @@
+# Distillation Algorithm Configuration
+distillation:
+    num_prompts_per_step: 128
+    num_generations_per_prompt: 1
+    max_rollout_turns: 1 # for multi-turn rollouts. Math Environments just have 1 turn (answering the question)
+    max_num_steps: 1000
+    val_batch_size: 64
+    val_period: 20
+    val_at_start: false
+    max_val_samples: 512
+    topk_logits_k: 64
+    seed: 42
+
+loss_fn:
+    temperature: 1.0
+    kl_type: "mixed" # forward, reverse, mixed
+    mixed_kl_weight: 0.5 # when kl_type is "mixed", this is the weight of the forward KL
+    zero_outside_topk: false # if true, tp should be 1: zero out the teacher logits outside the top k when calculate forward KL loss
+
+checkpointing:
+    enabled: true
+    checkpoint_dir: "checkpoints/distillation-${policy.model_name}"
+    metric_name: "val_reward"
+    higher_is_better: true
+    keep_top_k: 3
+    save_period: 10
+    checkpoint_must_save_by: null
+
+policy: &POLICY_BASE
+    model_name: "Qwen/Qwen3-1.7B-Base"
+    tokenizer:
+        name: ${..model_name} ## specify if you'd like to use a tokenizer different from the model's default
+    train_global_batch_size: 64
+    train_micro_batch_size: 1
+    generation_batch_size: 64
+    logprob_batch_size: 1
+    max_total_sequence_length: 8192
+    precision: "bfloat16"
+    logprob_chunk_size: null
+
+    dtensor_cfg: &DTENSOR_BASE
+        enabled: true
```

DTensor v2 (AutoModel) is supported and we are recommending to use it, can you add `_v2: true` here and also in other recipes?
it should be similar with v1 (which you are using now), but can you also take a run to make sure it can work well?

```suggestion
    dtensor_cfg: &DTENSOR_BASE
        _v2: true
        enabled: true
```

---

have set the default config with v2=true.

---

thanks for the update, and the recipes under `examples/configs/recipes/llm` also need update.

---

has updated all recipe config files.